apps/condo/domains/user/access/RegisterUserExternalIdentityService.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Generated by `createservice user.RegisterUserExternalIdentityService`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canRegister ({ args: { data: { user: { id: userId } } }, authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.type === RESIDENT && user.id === userId) {
```

Comment:
Should return true/false

Reply:
this is required to pass a filter object in return access method statement. That helps to avoid user retrieving identities own by other users

Reply:
changed